logbook-core/src/test/java/org/zalando/logbook/ByteStreamUtilsTest.java

```diff
@@ -0,0 +1,35 @@
+package org.zalando.logbook;
+
+/*
+ * #%L
+ * Logbook: Core
+ * %%
+ * Copyright (C) 2015 - 2016 Zalando SE
+ * %%
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * 
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * #L%
+ */
+
+import org.junit.Test;
+
+import static org.junit.Assert.assertTrue;
+
+public class ByteStreamUtilsTest {
+
+    @Test
+    public void shouldConstructorBePackageProtected() {
+        // just for coverage
+        assertTrue(new ByteStreamUtils() != null);
```

Comment:
I agree, it's why on other projects I'm working on we didn't fix coverage threshold at 100%.
Other option could be to use PowerMock to avoid repeating reflection stuff.